tailwind.config.js

```diff
@@ -11,9 +11,9 @@ module.exports = {
     },
     screens: {
       // Default: Mobile
-      'm-xxs': '350px', // Special case for progress bar: Extra extra small screen
-      'm-xs': '420px', // Special case for progress bar: Extra small screen
-      'm-s': '550px', // Special case for progress bar: Small screen
+      'm-xxs': '350px', // Special case for responsivity: Extra extra small screen
+      'm-xs': '420px', // Special case for responsivity: Extra small screen
+      'm-s': '550px', // Special case for responsivity: Small screen
```

Comment:
Kändes dumt att spec:a att detta bara gällde progress bar då dessa kommer in i bilden nu framöver också

Reply:
Borde det inte vara "for react"? Alla screens-värden är ju för responsivity, så i nuläget är kommentaren lite överflödig

Reply:
Jag funderar också på fnuttarna, vad betyder de här? varför har t och d inte fnuttar?

Reply:
1. Kan ändra kommentarerna.

2. För att när man använder bindestreck så kan inte de deklareras utan fnuttar. Att vi kör med bindestreck är för att de kört med de för andra deklarationer

Reply:
Rimligt!